### ISSUE TITLE
Marlin 2.0 reports M420 in capitals

### ISSUE THREAD

**Issue by mvn23** [Xennis/OctoPrint-RestoreLevelingAfterG28#13]
Bed leveling status is not detected properly on Marlin 2.0.5.4 (Ender 3 Pro).
The report looks like this:
```
Send: M420 V
Recv: Mesh Bed Level data:
Recv: 3x3 mesh. Z offset: 0.00000
Recv: Measured points:
Recv:         0        1        2
Recv:  0 -0.02500 +0.07500 +0.10000
Recv:  1 +0.00000 +0.07500 +0.00000
Recv:  2 -0.02500 +0.05000 +0.05000
Recv: 
Recv: echo:Bed Leveling ON
Recv: echo:Fade Height OFF
Recv: ok
```
Notice the `ON` and `OFF` in capitals, while the plugin is looking for `On` only:
https://github.com/Xennis/OctoPrint-RestoreLevelingAfterG28/blob/8c91aa4b4804250c3c91e4683eefe0d1ae5abe7b/octoprint_restorelevelingafterg28/__init__.py#L42
If I change this to `self.leveling_enabled = bool("Bed Leveling ON" in line)` it works as expected, but a more thorough fix is necessary to support both reporting methods.

**Comment by Xennis** [comment]
Hey @mvn23 thanks a lot for reporting and already investigating this issue. I addressed it in #14 (including adding tests for it). 

I've released that change in [1.1.0](https://github.com/Xennis/OctoPrint-RestoreLevelingAfterG28/releases/tag/1.1.0).

If there are further issues, let me know.

Regards, Xennis

**Comment by mvn23** [comment]
Awesome! Thanks for the feedback!
